frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -33,8 +32,10 @@ use sp_npos_elections::{
 use sp_runtime::{offchain::storage::StorageValueRef, traits::TrailingZeroInput, SaturatedConversion};
 use sp_std::{cmp::Ordering, convert::TryFrom, vec::Vec};
 
-/// Storage key used to store the persistent offchain worker status.
-pub(crate) const OFFCHAIN_LOCK: &[u8] = b"parity/multi-phase-unsigned-election";
+/// Storage key used to store the last block number at which offchain worker ran.
+pub(crate) const OFFCHAIN_LAST_BLOCK: &[u8] = b"parity/multi-phase-unsigned-election";
```

Comment:
```suggestion
pub(crate) const OFFCHAIN_LAST_BLOCK: &[u8] = b"parity/multi-phase-unsigned-election/block";
```
nit; this doesn't seem like something broad enough to take the top-level key.

Reply:
agree, for now I kept it because I am not sure what happens to the old value, and it would probably linger into the storage until eternity or sth, albeit being small.